homework1.0/src/main/java/homework04/treeSet/Main.java

```diff
@@ -0,0 +1,41 @@
+package homework04.treeSet;
+
+/**
+ * You need to store Persoane in a TreeSet.
+ * Define 2 Comparators (one for name and one for age) that will be used when creating the TreeSet;
+ * Add Persoane to the TreeSet; iterate throug the treeset and print the name and the age;
+ *
+ * @author Loránd
+ */
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
Ai rezolvat problema in main. De ce?

Cum testezi acel main? cum il refolosesti?
NU mai rezolva in main ci in clase de sine statatoare pe care le si testezi apoi.